### ISSUE TITLE
[install] Use the Xorg session by default

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/winesapOS#674]
Now that we install the Wayland session for KDE Plasma, it is the default session. It is considered experimental in Plasma 5 and won't be stable until Plasma 6. That is slated to come out in February of 2024.

Some applicantions have issues on Wayland. For example, the ClamTk GUI seems to work but trying to run an ant-virus scan silently crashes the entire program. Logs show this is because of a Wayland related crash. Switching to Xorg fixes it.

**Comment by LukeShortCloud** [comment]
Wayland with the proprietary graphics drivers is still unstable, too.

**Comment by LukeShortCloud** [comment]
LightDM stores a configuration file to `~/.dmrc` of which was the last session selected.

Example for Plasma and Wayland:

```
[Desktop]
Session=plasamawayland
```

Example for Plasma and Xorg:

```
[Desktop]
Session=plasma
```

**Comment by LukeShortCloud** [comment]
Now that we have Open Gamepad UI and its session installed, winesapOS is defaulting to the Gamescope session for it. Gamescope will not work on all hardware (NVIDIA, specifically).

I tried using the `~/.dmrc`. Although the file update the last used session whenever a user logs in (only if the file exists), it is not honored anymore by modern versions of GNOME Display Manager (GDM) or LightDM. Using the `/etc/lightdm/lightdm.conf` file also only seems to work for auto login. We do have autologin but it takes 30 seconds before we log in (in case a user needs time to switch sessions). Impatient folks will simply click "Login" and drop into a different session. Instead, I found this to work for our needs:

```
$ sudo vim /var/lib/AccountsService/users/winesap
[User]
Language=
XSession=plasma
```

https://superuser.com/questions/685970/how-to-set-a-default-desktop-environment-at-system-start
https://help.gnome.org/admin/system-admin-guide/stable/session-user.html.en
